crates/lang/codegen/src/generator/events.rs

```diff
@@ -90,6 +90,7 @@ impl<'a> Events<'a> {
         quote! {
             #[allow(non_camel_case_types)]
             #[derive(::scale::Encode, ::scale::Decode)]
+            #[cfg(not(target_os = "__ink_EventBase"))]
```

Comment:
What alternatives are there to this, what about `#[cfg(not(feature = "__ink_EventBase"))]`. 

Would be nice to have custom tool attributes e.g. https://rust-lang.github.io/rfcs/2103-tool-attributes.html, but maybe could piggyback on a `rustfmt` or `clippy` one?

Reply:
>What alternatives are there to this, what about `#[cfg(not(feature = "__ink_EventBase"))].`

That would work too, I'll change it.

>maybe could piggyback on a `rustfmt` or `clippy` one?

Makes us dependent on those though. I had the same idea at first and was looking at `clippy` attributes, but AFAICT those are only enums, so we can't encode ink!-specific info.

Reply:
Btw. I hadn't tried `feature` since I had concerns that this might require the feature definition somewhere. But from testing locally it seems to work just fine.